src/containers/month.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TaskGroup from './task-group';
+
+const Month = props => (
+  <li key={props.abbr} className='months-list__item'>
+    <div className='months-list-item__label-container'>
+      <span className='months-list-item__label' onClick={() => this.props.onMonthSelect}>
```

Comment:
onClick={this.props.onMonthSelect} should work provided that particular property has the data type of function. This may well change once I've actually written a method for it to call though :)